packages/pie-toolbox/src/code/drag/placeholder.jsx

```diff
@@ -74,18 +74,17 @@ const styles = (theme) => ({
     backgroundColor: `${grey[300]}`,
   },
   board: {
-    border: '1px solid #D1D1D1',
     padding: theme.spacing.unit,
     display: 'flex',
     flexWrap: 'wrap',
     alignItems: 'center',
-    minHeight: '200px',
+    minHeight: '100px',
     justifyContent: 'center',
     overflow: 'hidden',
     touchAction: 'none',
+    backgroundColor: '#ECEDF1',
```

Comment:
I introduced this new color in our colors scheme.
Let's use it.
https://github.com/pie-framework/pie-lib/blob/develop/packages/pie-toolbox/src/code/render-ui/color.js#L71